packages/fork-choice/test/unit/protoArray/protoArray.test.ts

```diff
@@ -47,11 +51,13 @@ describe("ProtoArray", () => {
       parentRoot: unknown,
       stateRoot,
       targetRoot: finalizedRoot,
-      executionPayloadBlockHash: null,
+
       justifiedEpoch: genesisEpoch,
       justifiedRoot: stateRoot,
       finalizedEpoch: genesisEpoch,
       finalizedRoot: stateRoot,
+
+      ...{executionPayloadBlockHash: null, executionStatus: ExecutionStatus.PreMerge},
```

Comment:
Why spread a new object instead of assigning directly?

Reply:
umm... typescript was giving error as assigning directly might not be helping it identify the union conditions

Reply:
Oh makes sense :+1: